docs/usage/modules/.pages

```diff
@@ -1,3 +1,6 @@
 title: Renovate Modules
+nav:
+  - Introduction: index.md
+  - ...
```

Comment:
Is this on purpose?

Reply:
yes, so introduction is first page and then the rest is generated 😏